test/e2e/kubernetes/namespace/namespace.go

```diff
@@ -41,8 +41,18 @@ func CreateIfNotExist(name string) (*Namespace, error) {
 	n, err := Get(name)
```

Comment:
Can we rename this function to `CreateDeleteIfExists` to reflect the new behavior?

Reply:
Sure, let me rename it.